ViewModel/ImpactAreaScenario/IASElementSet.cs

```diff
@@ -20,6 +20,9 @@ public class IASElementSet : ChildElement
 
         private string _Description = "";
         private int _AnalysisYear;
+
+        private List<metrics.Results> _Results = new List<metrics.Results>();
```

Comment:
Namespacing

Reply:
it won't let me change this one. I think another class might be called Results or something.